components/src/components/SplitButton/SplitButton.stories.tsx

```diff
@@ -86,12 +92,14 @@ export const Overview = (args: SplitButtonProps) => (
       primaryAction={{ label: 'Tekst', icon: PlusCircledIcon }}
       secondaryActions={items}
       size="tiny"
+      purpose="secondary"
     />
     <SplitButton
       {...args}
       primaryAction={{ label: 'Tekst', loading: true }}
       secondaryActions={items}
       size="tiny"
+      purpose="danger"
     />
   </StoryTemplate>
 );
```

Comment:
eksemplene her skal vise ulik innhold i knappen: tekst, ikon og tekst, loading. for å vise nye varianter lag ny rad og behold den første raden slik den var originalt. da blir det 1 rad for primary og 1 for secondary. begge skal ha samme eksempler for innhold i knappen.

Som nevnt lengre ned skal det ikke være `danger`.

Reply:
Syns det blir ekstremt mange komponenter hvis man skal duplisere alt - så prøvde meg på en liten vri for å generere nye visninger.

Reply:
kjempebra 👍